Packages/ClientRuntime/Sources/Networking/ClientError.swift

```diff
@@ -15,6 +15,7 @@ public enum ClientError: Error, Equatable {
     case unknownError(String)
     case authError(String)
     case retryError(Error)
```

Comment:
What purpose/role does `retryError` play now that we have `maxmimumRetryAttemptsError`?